### DISCUSSION TITLE
Milvus with LDAP

### DISCUSSION THREAD

**Discussion by MR-GOYAL** [milvus-io/milvus#32571] [Q&A and General discussion]
I want to use Milvus with LDAP but it seems Milvus doesn't support LDAP right. Is there any future plan to do that. If not how can i work on that?
In this [discussion](https://github.com/milvus-io/milvus/discussions/26455), they have suggested to integrate a hook. Can anyone give me brief  How to do that?

**Comment by xiaofan-luan** [comment]
I think it's too heavy to support ldap in milvus, implement a service to migrate user in your ldap server to milvus should not be hard.

But if you do want to implement one, the contribution is highly welcomed. 
Something like https://dev.mysql.com/doc/refman/8.0/en/ldap-pluggable-authentication.html

the LDAP is only valid to verify username password

**Comment by xiaofan-luan** [comment]
I'm not an expert of ldap but https://github.com/go-ldap might be good to start with.
We need to integrate it with milvus authentication

**Reply by MR-GOYAL** [reply]
is there any doc or blog on milvus authentication ?
Does milvus provide any Interface to extend existing authentication module?
